pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -291,6 +291,7 @@ func NewContainerManager(mountUtil mount.Interface, cadvisorInterface cadvisor.I
 			nodeConfig.ExperimentalCPUManagerReconcilePeriod,
 			machineInfo,
 			cm.GetNodeAllocatableReservation(),
+			cpumanager.DefaultCPUManagerStateFilePath,
```

Comment:
i prefer that we pass the kubelet RootDirectory as input, and require the cpumanager to store its state relative to that path.

Reply:
ok